tests_integration/__tests__/__snapshots__/cursor-offset.js.snap

```diff
@@ -20,13 +20,10 @@ exports[`cursorOffset should not be affected by full-width character (stdout) 1`
 exports[`cursorOffset should not be affected by full-width character (write) 1`] = `Array []`;
 
 exports[`write cursorOffset to stderr with --cursor-offset <int> (stderr) 1`] = `
-"1
+"2
 "
 `;
 
-exports[`write cursorOffset to stderr with --cursor-offset <int> (stdout) 1`] = `
-"1;
-"
-`;
+exports[`write cursorOffset to stderr with --cursor-offset <int> (stdout) 1`] = `" 1"`;
```

Comment:
Not sure why these changed... @ikatyang could you spot-check?

Reply:
This looks like the correct change:

- with `babylon`:

  before (offset=2)

  ```
   1|
  ```

  after (offset=1)

  ```
  1|;
  ```

- with `raw`:

  before (offset=2)

  ```
   1|
  ```

  before (offset=2)

  ```
   1|
  ```

Reply:
This looks like the correct change:

- with `babylon`:

  before (offset=2)

  ```
   1|
  ```

  after (offset=1)

  ```
  1|;
  ```

- with `raw`:

  before (offset=2)

  ```
   1|
  ```

  before (offset=2)

  ```
   1|
  ```